src/themes/theme.ts

```diff
@@ -1,6 +1,6 @@
 import * as React from "react";
 
-export type Variant = "primary" | "secondary" | "danger" | "warning" | "info" | string;
+export type Variant = "default" | "primary" | "success" | "info" | "warning" | "danger"| string;
```

Comment:
Hmm... it seems that `"default"` was missing indeed. `"secondary"` was never used, so that's also OK, but the `bs5` theme uses `"secondary"` as a default in the `mapAnyVariant()`. Is the default unnecessary in `mapAnyVariant()`?

Reply:
I'm not sure if I understand the question. I added `default` here as an option because it is a bootstrap 3 variant (and secondary is not?) and I guess someone could use it.

Reply:
Sorry for bad phrasing, took me also a while to understand what I tried to communicate here. Anyways, it seems like I didn't understand that bs5 doesn't have a variant named "default" anymore. So you can disregard this discussion, the code changes are OK